node/status_node.go

```diff
@@ -551,3 +565,14 @@ func (n *StatusNode) ensureSync(ctx context.Context) error {
 		}
 	}
 }
+
+// Discover sets up the discovery for a specific topic.
+func (n *StatusNode) Discover(topic string, max, min int) (err error) {
```

Comment:
Should we pass `params.Limit` instead of `max` and `min`?

Reply:
I've moved it here to avoid adding it as dependency on the `peer service` to keep it as simple as possible.